sdks/java/expansion-service/build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
   implementation project(path: ":sdks:java:core", configuration: "shadow")
   implementation project(path: ":runners:core-construction-java")
   implementation project(path: ":runners:java-fn-execution")
+  implementation project(path: ":sdks:java:extensions:google-cloud-platform-core")
```

Comment:
I would like to avoid pulling in additional dependencies to expansion service since it will boat the jar. Why was this needed ?

Reply:
Also it could create conflicts for other expansion service jars. Can we use GCP expansion service instead ?

https://github.com/apache/beam/tree/master/sdks/java/io/google-cloud-platform/expansion-service

Reply:
The class `GcpOptions` is in that module. If this is unnecessary please let me know, but I thought adding this as a dependency was necessary to import GcpOptions.

Reply:
Is see, in that case think we would need more refactoring to do this without unnecessarily bloating all expansion service jars.

For example, we could consider introducing a method to "ExpansionService" called "configureExpansionServiceProperties" (a no-op by default) and implement that in a "ExpansionService" sub-class available in "google-cloud-platform/expansion-service/"

Reply:
Done, refactored so that only the GCP expansion service receives this pipeline option, and only that jar has increased in size.